tools/bbbsupervisor/bbbsupervisor.go

```diff
@@ -486,23 +490,46 @@ func handleBitcoindIDB(event watcherEvent, pState *supervisorState) error {
 				return fmt.Errorf("Handling trigger %s: Initial set. Setting BBB config value `false` failed: %v", event.trigger.String(), err)
 			}
 		}
-		pState.prometheusLastStateIBD = newValue // set the initial value for the state
-	} else if oldValue == 1 && newValue == 0 { // IDB finished
-		err := setBBBConfigValue("bitcoin_ibd", "false")
+		pState.prometheusLastStateIBD = isActive // set the initial value for the state
+	} else if wasActive == 1 && isActive == 0 { // IBD finished
+		// unset clearnet IBD redis key when the IBD is finished
+		err := unsetClearnetIDB()
+		if err != nil {
+			return fmt.Errorf("Handling trigger %s: %s", event.trigger.String(), err)
+		}
+		err = setBBBConfigValue("bitcoin_ibd", "false")
 		if err != nil {
 			return fmt.Errorf("Handling trigger %s: setting BBB config value to `false` failed: %v", event.trigger.String(), err)
 		}
-		pState.prometheusLastStateIBD = newValue
-	} else if oldValue == 0 && newValue == 1 { // IDB (re)started
+		pState.prometheusLastStateIBD = isActive
+	} else if wasActive == 0 && isActive == 1 { // IBD (re)started
 		err := setBBBConfigValue("bitcoin_ibd", "true")
 		if err != nil {
 			return fmt.Errorf("Handling trigger %s: setting BBB config value to `true` failed: %v", event.trigger.String(), err)
 		}
-		pState.prometheusLastStateIBD = newValue
+		pState.prometheusLastStateIBD = isActive
 	}
 	return nil
 }
 
+// unsetClearnetIDB unsets (0 - download blocks over Tor) the ibdClearnetRedisKey if set.
+// The key can only be set back to 1 (download blocks over clearnet) via RPC.
+func unsetClearnetIDB() (err error) {
+	const ibdClearnetRedisKey string = "bitcoin:ibd-clearnet"
+	isIBDClearnet, err := getRedisInt(ibdClearnetRedisKey)
+	if err != nil {
+		return fmt.Errorf("getting redis key %s failed: %v", ibdClearnetRedisKey, err)
+	}
+	if isIBDClearnet == 1 {
+		log.Printf("IDB finished. Setting %s to %d.\n", ibdClearnetRedisKey, 0)
+		err = setRedisValue(ibdClearnetRedisKey, "0")
```

Comment:
My bad, it's not enough that the Redis key is set to `0`, also the bitcoind config file must be changed and the service needs to be restarted. The supervisor should call the following command, which also sets the Redis value back to zero.
```
bbb-config.sh disable bitcoin_ibd_clearnet
```
Sorry about the sloppy spec... 😶